app/src/main-process/main.ts

```diff
@@ -104,6 +105,15 @@ if (__WIN32__ && process.argv.length > 1) {
   }
 }
 
+if (__DARWIN__) {
```

Comment:
Yeah, but I wouldn't worry too much about deadcoding it. In theory the `if (false && ...)` should be deadcoded since it could never be true. But even if it's not, I'm not too worried about it.